.github/actions/godot-cpp-build/action.yml

```diff
@@ -2,7 +2,7 @@ name: Build godot-cpp
 description: Build godot-cpp with the provided options.
 
 env:
-  GODOT_CPP_BRANCH: 4.3
+  GODOT_CPP_BRANCH: 4.4
```

Comment:
@dsnopek I assume it's intentional that we stay one branch behind? Or should this target godot-cpp `master`?

Reply:
Yes, it's intentional to stay one version behind - it prevents circular dependencies between Godot and godot-cpp PRs that affect the GDExtension interface.

So, the change you have here is correct!

Reply:
Sounds good! I guess I just need you to push the `4.4` branch to godot-cpp before merging this?

Reply:
Hm. I just (like a couple minutes ago) made the godot-cpp release and `4.4` branch, which I think would have been after CI ran here, but CI didn't fail, which I would have expected. :-/

Either we had perfect timing or the CI isn't doing what it should for godot-cpp? Looking at the logs and trying to figure it out now..

Reply:
Looking at the log, it used godot-cpp commit https://github.com/godotengine/godot-cpp/commit/06082d77278e8f444db27e0944aab5f309d4fc72 which was `master` _before_ I pushed the 4.4 release stuff. Maybe it falls back on `master` if it can't find `4.4`? Anyway, I've queued the CI job to run again, although, I guess that doesn't tell us much because `4.4` and `master` point at the same commit at the moment.

I think we'll need to wait until there's more commits on godot-cpp `master` to verify if Godot's CI is really using `4.4` or not. And if it does start using `4.4`, I guess it just means there's a fallback to `master` when `4.4` doesn't exist, rather than a failure

Reply:
VSCode is telling me that `env` isn't allowed here:

![Selection_279](https://github.com/user-attachments/assets/715dd798-4d8e-4a45-ae04-05bd69a32482)

Maybe `env` doesn't go in actions, but only in workflows? We might need move this into `inputs` or something? I'm not the most knowledgable on GitHub Actions

Reply:
Maybe something like PR https://github.com/godotengine/godot/pull/103514

Let's see what happens with that PR when CI runs :-)